src/stylesheets/settings/_mixins.scss

```diff
@@ -215,3 +204,73 @@
     }
   }
 }
+
+/*--- Heading Typography Mixins ---*/
+
+@mixin alpha {
```

Comment:
These mixin names (to me anyway) mean absolutely nothing. Is it worth giving them a more descriptive name?

Reply:
These are pretty common sources of technical symbols and labels, also used as entities for HTML, CSS content and JS. 

CSS example - `list-style: lower-alpha`
HTML entity example - `&Alpha; &Beta;`

Reply:
Ah, coolio.